README.md

```diff
@@ -13,15 +13,17 @@ draft](https://tools.ietf.org/html/draft-yasskin-wpack-use-cases)).
 
 <!-- TOC -->
 
-- [Explainers](#explainers)
```

Comment:
IMHO the table of contents does not need to have a reference to the README's title.

Reply:
This got added automatically on save. Should I still remove it?

Reply:
Hmm, interesting, I would still remove it.